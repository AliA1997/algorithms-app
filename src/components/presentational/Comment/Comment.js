import React from 'react';

export const Comment = () => {
    return (
        <div>
            Comment
        </div>

    );
};

export default Comment;