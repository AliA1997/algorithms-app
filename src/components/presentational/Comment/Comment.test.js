//import commponent to test and react
import React from 'react';
import { Comment } from './Comment';
//import test rendere
import render from 'react-test-renderer';

describe('Test if it renders correctly', () => {
    // beforeAll(() => {
    //     const tree = render.create(<Comment />).toJSON();
    // })
    test('Test if it renders correctly', () => {
        const tree = render.create(<Comment />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})