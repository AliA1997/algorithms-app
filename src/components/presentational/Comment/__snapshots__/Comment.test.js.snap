// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if it renders correctly Test if it renders correctly 1`] = `
<div>
  Comment
</div>
`;
